src/Hosting/Hosting/src/Startup/ConventionBasedStartup.cs

```diff
@@ -23,13 +23,9 @@ public void Configure(IApplicationBuilder app)
         {
             _methods.ConfigureDelegate(app);
         }
-        catch (Exception ex)
+        catch (Exception ex) when (ex is TargetInvocationException)
```

Comment:
Is this interestingly different from `catch (TargetInvocationException ex)`?

Reply:
No, don't know why I didn't simplify it further. Fixed now.